modules/glb/README.md

```diff
@@ -1,29 +1,26 @@
-# Module CIS GLB
+# CIS global load balancer (GLB) module
 
-This module is used to provision Global Load Balancers, Origin pools and monitors.
+This module is used to provision global load balancer that include load balancers, origin pools and health checks.
 
-## Example Usage
 
-```terraform
-module "cis_glb" {
-  cis_id             = var.cis_id
-  domain_id          = var.domain_id
-  source             = "../../modules/glb"
-  glb_name           = "cis_glb"
-  fallback_pool_name = "cis_fpn"
-  region_pools       = local.region_pools
-  origin_pools       = local.origin_pools
-  monitors           = local.monitors
-}
+When `glb_proxied` is set as `true`, then `ttl` is automatically set and cannot be updated.
 
+This means that when you run a `terraform plan` command after a successful `terraform apply`, the output shows that the ttl value requires an update, as shown in the following example. However, your infrastructure will not be affected.
```

Comment:
Nitpick: I suggest either using the abbreviation (TTL) or putting the variable name in backticks (`ttl`). Might have missed this earlier.

Reply:
sure, made the changes as `ttl`